examples/destination_adobe_analytics.tf

```diff
@@ -0,0 +1,9 @@
+resource "rudderstack_destination_adobe_analytics" "example" {
+    name = "my-adobe-analytics"
+
+    config {
+        tracking_server_url = "http://sampleurl.com"
```

Comment:
The practice we are following for the rest of destinations is to also include optional config fields in the example, but keep them commented out. This is in order to give an example to the user of how they look.